server/.eslintrc.js

```diff
@@ -1,10 +1,25 @@
 module.exports = {
-	extends: ["@codeyourfuture/standard"],
-	parserOptions: {
-		ecmaVersion: 2017,
-	},
-	env: {
-		node: true,
-		es6: true,
-	},
+  extends: [
+    "@codeyourfuture/standard",
+    "eslint:recommended",
+    "plugin:import/errors",
+    "plugin:jsx-a11y/recommended",
+  ],
+  rules: {
+    "react/prop-types": 0,
+    "no-console": 1,
+  },
+  plugin: ["react", "import", "jsx-a11y"],
```

Comment:
I'm not sure why you want this plugins and react rules. we don't have any react in the server